src/include/mallocMC/creationPolicies/Scatter.hpp

```diff
@@ -330,11 +330,14 @@ namespace mallocMC
                 uint32 chunksize) -> uint32
             {
                 if(fullsegments != 32)
-                    return alpaka::math::max(
-                               acc,
-                               0u,
-                               (int) pagesize - (int) fullsegments * segmentsize - (int) sizeof(uint32))
-                        / chunksize;
+                    return alpaka::math::min(
+                        acc,
+                        31,
```

Comment:
Could this `31` (and/or the `32` above) become a constexpr variable with some telling name?

Reply:
I will do it separate PR.